chimeraTKApp/src/ChimeraTK/EPICS/ArrayRecordDeviceSupport.h

```diff
@@ -116,11 +116,14 @@ class ArrayRecordDeviceSupportTrait : public RecordDeviceSupportBase {
   /**
    * Updates the record's TIME field with the specified time stamp.
    */
-  void updateTimeStamp(TimeStamp const &timeStamp) {
+  void updateTimeStamp(VersionNumber const &versionNumber) {
+    auto time = versionNumber.getTime();
+    double timeInNanosecs = std::chrono::time_point_cast<std::chrono::nanoseconds>(time).time_since_epoch().count();
+    uint64_t secs = std::floor(timeInNanosecs / 1.e9);
```

Comment:
I prefer the fully-qualified form whererever possible, so `uint64_t` should be `std::uint64_t`. Anyway, I wonder whether `std::int64_t` might be better, because to my knowledge time-stamps are signed and might be negative when representing a point in time before epoch.

Reply:
To be consistent, I will change this to std::chrono::seconds::rep